packages/perseus-score/src/widgets/free-response/validate-free-response.test.ts

```diff
@@ -0,0 +1,18 @@
+import validateFreeResponse from "./validate-free-response";
+
+describe("free response validation", () => {
+    it("should return invalid for empty user input", () => {
+        const result = validateFreeResponse({currentValue: ""});
+        expect(result).toHaveInvalidInput();
```

Comment:
I would recommend adding an expected validation message (which is just an error code, like I mentioned in the other comment):

```suggestion
        expect(result).toHaveInvalidInput("USER_INPUT_EMPTY");
```

See `perseus/packages/perseus-score/src/widgets/categorizer/validate-categorizer.test.ts`